packages/react-native/Libraries/Blob/URL.js

```diff
@@ -18,9 +17,9 @@ if (
   NativeBlobModule &&
   typeof NativeBlobModule.getConstants().BLOB_URI_SCHEME === 'string'
 ) {
-  const constants = NativeBlobModule.getConstants();
   // $FlowFixMe[incompatible-type] asserted above
   // $FlowFixMe[unsafe-addition]
+  const constants = NativeBlobModule.getConstants();
```

Comment:
Why was this moved?

I understand that `$FlowFixMe` directives were related to the line below?

Reply:
resolved